### PR TITLE
prepare for 0.12.20+9

### PR DESCRIPTION
**Pull request by grouma**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.12.20+9
+
+* Introduce new flag --chain-stack-traces to conditionally chain stack traces.
```
Comment (kevmoo) [inline CHANGELOG.md]: use ticks to make `--chain-stack-traces` pretty

Comment (grouma) [inline CHANGELOG.md]: Done.

**Comment by nex3** [comment]
This adds a new feature—why was it released as a patch version?

**Comment by kevmoo** [comment]
User error, I'm guessing.

**Comment by grouma** [comment]
My apologies. When we sync up lets review what the best practices and preferred patterns for this repo so I don't make these mistakes in the future.

**Pull request merged**